R/main.R

```diff
@@ -14,7 +14,8 @@ Options:
   --go-signal=PATH               Relative path for go signal
   --queue-id=ID                  rrq ID
   --workers=WORKERS              Number of workers to spawn [default: 0]
-  --backup-period=BACKUP_PERIOD  How frequently should backup be run, 0 or negative for no backup [default: 600]"
+  --backup-period=BACKUP_PERIOD  How frequently should backup be run, 0 or negative for no backup [default: 600]
+  --log-level=LEVEL              Detail level of logs info or trace [default: info]"
```

Comment:
Is this right? The levels are info and trace?

Reply:
yes, though there others: https://s-fleck.github.io/lgr/articles/lgr.html#log-levels